modules/common/networking/hosts.nix

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  ...
+}: let
+  # please note that .100. network is not
+  # reachable from ghaf-host. It's only reachable
+  # guest-to-guest.
+  # Use to .101. (debug) to access guests from host.
+  # debug network hosts are post-fixed: <hostname>-debug
+  ipBase = "192.168.100";
+  debugBase = "192.168.101";
+  hostsEntries = [
+    {
+      ip = 1;
+      name = "net-vm";
+    }
+    {
+      ip = 2;
+      name = "ghaf-host";
+    }
+    {
+      ip = 3;
+      name = "gui-vm";
+    }
+    {
+      ip = 4;
+      name = "ids-vm";
+    }
+    {
+      ip = 5;
+      name = "audio-vm";
```

Comment:
I think earlier the name address of each vms ended with .ghaf (for example: audio-vm.ghaf). In some vm config that use audio (chromium and element) there is pulseaudio config that refer to audio-vm.ghaf:4713. If the .ghaf ending addresses don't work anymore those configs should be updated to reflect the new name.

Reply:
.ghaf network references should be fixed now.